packages/tools/examples/stackAnnotationTools/index.ts

```diff
@@ -209,62 +181,10 @@ async function run() {
   // Init Cornerstone and related libraries
   await initDemo();
 
-  // Add tools to Cornerstone3D
-  cornerstoneTools.addTool(LengthTool);
-  cornerstoneTools.addTool(ProbeTool);
-  cornerstoneTools.addTool(RectangleROITool);
-  cornerstoneTools.addTool(EllipticalROITool);
-  cornerstoneTools.addTool(CircleROITool);
-  cornerstoneTools.addTool(BidirectionalTool);
-  cornerstoneTools.addTool(AngleTool);
-  cornerstoneTools.addTool(CobbAngleTool);
-  cornerstoneTools.addTool(ArrowAnnotateTool);
-  cornerstoneTools.addTool(PlanarFreehandROITool);
-  cornerstoneTools.addTool(EraserTool);
-  cornerstoneTools.addTool(KeyImageTool);
-
   // Define a tool group, which defines how mouse events map to tool commands for
   // Any viewport using the group
   const toolGroup = ToolGroupManager.createToolGroup(toolGroupId);
```

Comment:
I believe this example serves as a great starting point for new people joining the project. While advanced examples can be abstracted with these new utiltiies, keeping this one straightforward with the verbose API can help newcomers see how things work. Let's maintain this simplicity for basic examples like this and only refactor the advacned ones. Can you revert these chanages?

Reply:
Reverted as requested.  Moved the example to videoTools